arangod/RestServer/Metrics.h

```diff
@@ -93,52 +93,74 @@ template<typename T> class Gauge : public Metric {
   Gauge(T const& val, std::string const& name, std::string const& help,
         std::string const& labels = std::string())
     : Metric(name, help, labels) {
-    _g.store(val);
+    _g.store(val, std::memory_order_relaxed);
```

Comment:
You could put this into the initializer list instead of using a separate store:
`: Metric(name, help, labels), _g(val) {}`